benchmarks/advection/drop_supg.prm

```diff
@@ -1,7 +1,7 @@
 # Dropping box benchmark on the right half of the domain
 
 set Dimension                              = 2
-set Use years in output instead of seconds = false
+set Use years instead of seconds = false
```

Comment:
Might be nice to get the spaces back in so that the equals signs remain aligned.